grammar.js

```diff
@@ -45,7 +51,7 @@ module.exports = grammar({
 
     inline_verbatim: $ => seq($.verb_token, $.verb_delim, $.verb_body, $.verb_delim),
 
-    verb_token: $ => seq($._escape, 'verb', optional($._whitespace)),
+    verb_token: $ => seq($._escape, 'verb', optional('*')), // whitespace is _not_ gobbled here
```

Comment:
Any reason not to use `/verb\*?/`? I think that is slightly more readable and allows for new tokens to be added easily.

Reply:
No reason. I have run into trouble where TS prefers explicit strings over regex, but that's not the case here.

Reply:
TS seems to treat regex as very greedy, but unless it breaks something I tend to like them.